src/Tribe/Main.php

```diff
@@ -275,6 +275,7 @@ public function load_localize_data() {
 				'today'           => esc_html__( 'Today', 'the-events-calendar' ),
 				'clear'           => esc_html__( 'Clear', 'the-events-calendar' ),
 			),
+			'registration_prompt' => __( 'There is unsaved attendee information. Are you sure you want to continue?', 'event-tickets' ),
```

Comment:
Should be tribe-common and not event-tickets here

Reply:
Also, I feel like adding this into common needs to be revisited, it might be that we need a filter in common and ET will add `registration_prompt` to the list from it's own side.

Reply:
There seem to be other text that is translated that use different text domains, not `tribe-common`, so I kept with that. But I totally agree we need to revisit this and see if we can separate these.